src/main/java/net/openhft/chronicle/bytes/BytesStore.java

```diff
@@ -55,48 +54,6 @@
 public interface BytesStore<B extends BytesStore<B, U>, U>
         extends RandomDataInput, RandomDataOutput<B>, ReferenceCounted, CharSequence {
 
-    @SuppressWarnings("deprecation")
-    @Override
-    default boolean compareAndSwapFloat(@NonNegative long offset, float expected, float value)
-            throws BufferOverflowException, IllegalStateException {
-        return compareAndSwapInt(offset, Float.floatToRawIntBits(expected), Float.floatToRawIntBits(value));
-    }
-
-    @SuppressWarnings("deprecation")
-    @Override
-    default boolean compareAndSwapDouble(@NonNegative long offset, double expected, double value)
-            throws BufferOverflowException, IllegalStateException {
-        return compareAndSwapLong(offset, Double.doubleToRawLongBits(expected), Double.doubleToRawLongBits(value));
-    }
-
-    @SuppressWarnings("deprecation")
-    @Override
-    default int addAndGetInt(@NonNegative long offset, int adding)
```

Comment:
have these just moved? Do we have to move things around in files? It makes archaeology so much harder

Reply:
I have moved them back.